qt_6.1.0_linux_desktop_gcc_64.BUILD

```diff
@@ -1,56 +1,22 @@
 load("@bazel_skylib//lib:paths.bzl", "paths")
-
-QT_LIBRARIES = [
-    ("core", "QtCore", "Qt6Core", []),
-    ("network", "QtNetwork", "Qt6Network", []),
-    ("widgets", "QtWidgets", "Qt6Widgets", [":qt_core", ":qt_gui"]),
-    ("quick", "QtQuick", "Qt6Quick", [":qt_gui", ":qt_qml"]),
-    ("qml", "QtQml", "Qt6Qml", [":qt_core", ":qt_network"]),
-    ("qml_models", "QtQmlModels", "Qt6QmlModels", []),
-    ("gui", "QtGui", "Qt6Gui", [":qt_core"]),
-    ("opengl", "QtOpenGL", "Qt6OpenGL", []),
-]
-
-[
-    cc_import(
```

Comment:
Since we're using `cc_library` for everything now, I merged the two targets into one.